lib/api/src/modules/stories.ts

```diff
@@ -208,6 +192,15 @@ const initStoriesApi = ({
             const { name } = group;
             const parent = index > 0 && soFar[index - 1].id;
             const id = sanitize(parent ? `${parent}-${name}` : name);
+            if (parent === id) {
+              throw new Error(
+                `
+Invalid kind with id === parentId: '${id}'
```

Comment:
Can we log a bit more info here? The `id` might not be super clear given we've sanitized away a lot of special chars.

Reply:
Perhaps something like `Invalid part ${name}, leading to id === parentId ('${id}'), inside kind '${kind}'`

Reply:
So in your example it would log:

```
Invalid part '>', leading to id === parentId ('something-docs'), inside kind 'Something <docs />'
```

Reply:
good call. fixed.